address_book/contacts/views.py

```diff
@@ -0,0 +1,38 @@
+from django.shortcuts import render, redirect
+from contacts.forms import ContactForm
+from django.core.mail import send_mail
+from django.conf import settings
+from contacts.models import Contact
+
+def contact_list(request):
+    contacts = Contact.objects.all()
+    return render(request, 'contact_list.html',{'contacts':contacts})
+
+
+def add_contact(request):
+    if request.method == 'POST':
+        form = ContactForm(request.POST)
+        if form.is_valid():
+            form.save()
+
+            name = form.cleaned_data['first_name'] + ' ' + form.cleaned_data['last_name']
+            phone = form.cleaned_data['phone_number']
+            address = form.cleaned_data['address']
+            email = form.cleaned_data['email']
+
+            email_content = f"Name: {name}\nPhone: {phone}\nAddress: {address}\nEmail: {email}"
+
+            # Send the email
```

Comment:
Please add the link from where did you take this code?

Reply:
I have added  reference in changed file